install.bat

```diff
@@ -471,6 +471,47 @@ local function get_visual_studio_directory()
 	return nil
 end
 
+local function get_visual_studio_directory_from_vswhere()
+	assert(type(vars.LUA_RUNTIME)=="string", "requires vars.LUA_RUNTIME to be set before calling this function.")
+	local major, minor = vars.LUA_RUNTIME:match('VCR%u*(%d+)(%d)$')
+	if not major then
+		print(S[[    Cannot auto-detect Visual Studio version from $LUA_RUNTIME]])
+		return nil
+	end
+	if tonumber(major) < 14 then
+		return nil
+	end
+	local program_dir = os.getenv('PROGRAMFILES(X86)')
+	if not program_dir then
+		return nil
+	end
+	local vswhere = program_dir.."\\Microsoft Visual Studio\\Installer\\vswhere.exe"
+	if not exists(vswhere) then
+		return nil
+	end
+	local f = io.popen('"'..vswhere..'" -products * -property installationPath')
+	if not f then return nil, "failed to run vswhere " end

```

Comment:
Can you append any error from `popen` here?

Reply:
Sure. Code changed.